convene-web/spec/tasks/release_after_build_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe "rake release:after_build" do
+  subject(:run_rake_task!) do
+    ConveneWeb::Application.load_tasks
+    Rake::Task["release:after_build"].reenable
+    Rake::Task["release:after_build"].execute
+  end
+
+  it "does not fail" do
+    expect { run_rake_task! }.not_to raise_error
```

Comment:
Since we're using `subject`, could we shrink this down to `it { is_expected.not_to raise_error }`? (I always get confused by how to use rspec in general when dealing with errors :-p)

Reply:
```suggestion
    it { is_expected.not_to raise_error }
```

Reply:
Looks like I can't use one-liner syntax here.
https://github.com/zinc-collective/convene/runs/2163178859